assets/queries/terraform/aws/autoscaling_groups_supply_tags/metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "ba48df05-eaa1-4d64-905e-4a4b051e7587",
+  "queryName": "Autoscaling Groups Supply Tags",
+  "severity": "LOW",
+  "category": "Encryption",
```

Comment:
This query does not fit this category. This should be in the "Availability" category.